src/Microsoft.Diagnostics.Runtime/src/Linux/ElfFile.cs

```diff
@@ -63,6 +63,29 @@ internal Reader VirtualAddressReader
             }
         }
 
+        /// <summary>
+        /// Returns the address of a module export symbol if found
+        /// </summary>
+        /// <param name="symbolName">symbol name (without the module name prepended)</param>
+        /// <param name="offset">symbol offset returned</param>
+        /// <returns>true if found</returns>
+        public bool TryGetExportSymbol(string symbolName, out ulong offset)
+        {
+            try
+            {
+                if (DynamicSection is not null && DynamicSection.TryLookupSymbol(symbolName, out ElfSymbol? symbol) && symbol is not null)
```

Comment:
Not something you need to fix in this checkin, but .TryLookupSymbol should not throw since the name starts with Try.  I need to remember to look at that later...

Reply:
DynamicSection.TryLookupSymbol should throw any exceptions.  There is a try/catch in it for IOExceptions (read failures).  The reason this function (TryGetExportSymbol) has a try/catch is that getting the dynamic section (DynamicSection) can throw those exceptions.  Maybe this property should catch the exceptions and return null.